src/BloomExe/Publish/PdfViewer.cs

```diff
@@ -85,7 +112,8 @@ public bool ShowPdf(string pdfFile)
 		public void Print()
 		{
 #if !__MonoCS__
-			if (Settings.Default.UseAdobePdfViewer)
+			var arc = _pdfViewerControl as AdobeReaderControl;
+			if (arc != null)
 			{
 				((AdobeReaderControl)_pdfViewerControl).Print();
```

Comment:
Does viewing always precede printing, so that if we get here we know that Adobe Reader is installed?